vue/components/ui/molecules/input-currency/input-currency.vue

```diff
@@ -28,7 +28,7 @@ export const InputCurrency = {
             default: () => ({})
         },
         value: {
-            type: String,
+            type: String | Number,

```

Comment:
why the or?

Reply:
To allow both types, similarly to what we already do in `input`